bin/test-integration

```diff
@@ -25,7 +25,12 @@ bin/apply-crds
 
 kubectl get customresourcedefinitions
 
-pkgs=$(go list ./... 2> /dev/null)
-deps=`echo ${pkgs} | tr ' ' ","`
-ginkgo --slowSpecThreshold=50 -cover -coverpkg "$deps" integration/
-
+#deps="code.cloudfoundry.org/cf-operator/cmd,code.cloudfoundry.org/cf-operator/pkg"
+ginkgo --slowSpecThreshold=50 --cover --coverpkg code.cloudfoundry.org/cf-operator/pkg/... integration/
+
+# Send code coverage results to coveralls.io
```

Comment:
Can we move this into a separate file? I run this script locally a lot.

Reply:
Hi @manno , sure, I was also thinking the same. I am not yet done. I am testing a few things. will be done by tonight.